index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+ 
+  <head>
+    <title>Kurt Cobain</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+   
+  </head>
+ 
+  <body id="homebg">
```

Comment:
Refrain from using id's for styling.  You can only have one per page, the specificity is hard to over come (https://css-tricks.com/specifics-on-css-specificity/) which will likely cause issues down the road, and you lose the benefit of CSS since you will need repeat styles for every element that could have just been styled once using the same class.  While not evil, it is not best practice and should only be used in certain edge cases (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/). 🔨